### PR TITLE
Hover on bg btns

### PR DESCRIPTION
**Pull request by loicgasser**
Add hover effect on bg buttons on desktop only.
[Demo](https://mf-geoadmin3.dev.bgdi.ch/gal_bg_hover/?dev3d=true)

### CODE REVIEW

File: src/components/backgroundselector/style/backgroundselector.less [hunk 1]
```diff
@@ -247,4 +250,7 @@
     background-color: gray;
     color: gray;
   }
+  &:hover {
+    background-color: gray;
+  }
```
Comment (oterral) [inline src/components/backgroundselector/style/backgroundselector.less]: Why not in 3d?

Comment (loicgasser) [inline src/components/backgroundselector/style/backgroundselector.less]: meant to put it under ga-disable3d

Comment (oterral) [inline src/components/backgroundselector/style/backgroundselector.less]: yes

**Comment by oterral** [comment]
lgtm thx @loicgasser

**Pull request merged**